test/setup

```diff
@@ -38,16 +38,14 @@ eof
 echo "deleting heat stack to destroy resources including VMs: ${VMS}"
 ${ROOT}/test/cleanup
 
-echo "creating heat template for test env ..."
```

Comment:
Have we adjusted ci-full and other old ci tasks that were still generating heat templates?